purchase_packaging/migrations/10.0.1.0.8/post-migrate.py

```diff
@@ -0,0 +1,9 @@
+import odoo
+
+
+def migrate(cr, version):
+    # We recompute possible void values
+    env = odoo.api.Environment(cr, odoo.SUPERUSER_ID, {})
+    void_purchase_lines = env['purchase.order.line'].search([
+        ('product_qty', '=', False)])
+    void_purchase_lines._compute_product_qty()
```

Comment:
you have to set this value in context to prevent recursivity: with_context(recomputing_product_qty=True)
see: https://github.com/OCA/stock-logistics-warehouse/pull/493/files#diff-0b094619c99f4fabbe358caff73c8626R205